### DISCUSSION TITLE
Is there a way to make a pywal-like wallpaper color generating in Colorblendr

### DISCUSSION THREAD

**Discussion by allhypr** [Mahmud0808/ColorBlendr#152] [General]
Bc i ported [`Catppuccin Mocha`](https://github.com/catppuccin/catppuccin), [`Tokyo Night`](https://github.com/tokyo-night/tokyo-night-vscode-theme), [`Decay`](https://github.com/decaycs/decaycs) and [`Yoru`](https://github.com/raexera/yoru?tab=readme-ov-file#art--colorscheme) color schemes into a ColorBlendr pallete, and i need to know if a dynamic pywal color scheme can be imitated with the saturation and luminosity bars or another method

**Comment by Mahmud0808** [comment]
I have no idea how pywal works or generates colors. I use [material-foundation/material-color-utilities](https://github.com/material-foundation/material-color-utilities) for ColorBlendr.

**Reply by hanubeki** [reply]
pywal picks multiple colors from the wallpaper for color palette.
Currently, ColorBlendr picks single color from the wallpaper right?

**Reply by Mahmud0808** [reply]
Yes, ColorBlendr creates palette based on one color.
